cli/lib/kontena/cli/plugins/install_command.rb

```diff
@@ -0,0 +1,27 @@
+require 'open3'
+
+module Kontena::Cli::Plugins
+  class InstallCommand < Clamp::Command
+
+    parameter 'NAME', 'Plugin name'
+
+    def execute
+      install_plugin(name)
+    end
+
+    def install_plugin(name)
+      plugin = "kontena-plugin-#{name}"
+      gem_bin = `which gem`.strip
```

Comment:
Not sure why we need the full path anyway. Couldn't we just use plain `gem`? `which` also would require to have the `gem` in path to get the full path, so IMO we do not gain much having the full path.